canonical/ubuntu-pro-client
# livepatch: better inform if command fails

## Why is this needed?
If the canonical-livepatch status command fails, we can mistakenly tell the users that Livepatch is disabled. This is an incorrect behavior as we should alert the users about the error. We are now reflecting that went wrong with the command on the output of pro status.

LP: #2019997

## Test Steps
1. Launch a Jammy LXD VM
2. Attach it to a Pro subscription
3. Verify that that there is a warning for Livepatch when running `pro status` (warning due to kernel incompatibility)
4. Switch to ubuntu user
5. Run `pro status` again and verify that we output an error running `canonical-livepatch status`

## Checklist


## Does this PR require extra reviews?
